Cannot pip install on Windows 7

- I downloaded the VC++ 9 compiler + build tools necessary to install Python 2.7 packages. 
- Tried `pip install annoy`

The build fails with the following: `error C3861: '__popcnt64': identifier not found`

```
c:<>\local\temp\pip-build-jmpbob\annoy\src\annoylib.h(350) : error C3861: '__popcnt64': identifier not found
          c:<>\local\temp\pip-build-jmpbob\annoy\src\annoylib.h(714) : see reference to function template instantiation 'T Hamming::distance<S,T>(const Hamming::Node<S,T> *,const Hamming::Node<S,T> *,int)' being compiled
          with
          [
              T=uint64_t,
              S=int32_t
          ]
          c:<>\local\temp\pip-build-jmpbob\annoy\src\annoylib.h(713) : while compiling class template member function 'uint64_t AnnoyIndex<S,T,Distance,Random>::get_distance(S,S)'
          with
          [
              S=int32_t,
              T=uint64_t,
              Distance=Hamming,
              Random=Kiss64Random
          ]
```

there are a whole lot of warnings on improper type conversion in between

```
 error: command 'C:<>\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\amd64\\cl.exe' failed with exit status 2

  ----------------------------------------
  Failed building wheel for annoy
```

- Python version:
```
c:\> python --version
Python 2.7.14 :: Anaconda, Inc.
```

--- a/src/annoylib.h
+++ b/src/annoylib.h
@@ -67,11 +67,9 @@ typedef unsigned __int64  uint64_t;
 
 #ifndef _MSC_VER
 #define popcount __builtin_popcountll
-#elif _MSC_VER == 1500
+#else // See #293, #358
 #define isnan(x) _isnan(x)
 #define popcount cole_popcount
-#else
-#define popcount __popcnt64
 #endif
 
 #ifndef NO_MANUAL_VECTORIZATION

